recipes/hydrodata/meta.yaml

```diff
@@ -1,67 +0,0 @@
-{% set name = "hydrodata" %}
```

Comment:
Please don't modify files outside of the recipe dir you are working on. Maybe you need to pull from master?

Reply:
This is already removed. I think you're looking at previous commits, not the latest one.
Do you mean that I close this PR an open a new one?

Reply:
maybe just open a new PR

Reply:
ok

Reply:
Should I mention you there or wait for someone to pick it up?